packages/flutter_tools/test/general.shard/build_system/targets/web_test.dart

```diff
@@ -310,6 +310,27 @@ void main() {
   }, overrides: <Type, Generator>{
     ProcessManager: () => MockProcessManager(),
   }));
+
+  test('WebReleaseBundle copies all files from web folder', () => testbed.run(() async {
+    fs.file(fs.path.join('web', 'index.html'))
+      ..createSync(recursive: true)
+      ..writeAsStringSync('hello');
+    fs.file(fs.path.join('web', 'test.js'))
+      ..createSync(recursive: true)
+      ..writeAsStringSync('alert(\'hello world\');');
+    fs.file(fs.path.join('web', 'abc.txt'))
+      ..createSync(recursive: true)
+      ..writeAsStringSync('hello world');
+    await const WebReleaseBundle().build(environment);
+
+    expect(environment.outputDir.childFile('test.js').existsSync(), true);
+    expect(environment.outputDir.childFile('test.js').readAsStringSync(),'alert(\'hello world\');');
+    expect(environment.outputDir.childFile('abc.txt').existsSync(), true);
+    expect(environment.outputDir.childFile('abc.txt').readAsStringSync(),'hello world');
+  }, overrides: <Type, Generator>{
+    ProcessManager: () => MockProcessManager(),
+    Platform: () => FakePlatform()..environment = <String, String>{'HOME': '/home/me'},
```

Comment:
This value seems overly specific. What config is depending on this? Could that be mocked instead?

Reply:
Which value do you exactly mean? The home directory? I took that one from some other tests for android. But I guess you mean something else?